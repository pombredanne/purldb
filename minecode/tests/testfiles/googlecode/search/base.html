


<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOINDEX,NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"profileUrl": null, "projectHomeUrl": null, "loggedInUserEmail": null, "projectName": null, "token": null, "assetHostPath": "http://www.gstatic.com/codesite/ph", "domainName": null, "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "relativeBaseUrl": ""};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 
 
 
 
 Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_list.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .labels a:link { color: #080 }
 .labels a:visited { color: #080 }
 .labels a:active { color: #f00 }
 .name { margin-top: 2ex; font-size: 120% }

 
 </style>
</head>
<body class="">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch&amp;followup=http%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 >
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="">
 <a href="/">
 
 <img src="http://www.gstatic.com/codesite/ph/images/code_small.png" alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 </td>
 <td style="white-space:nowrap;">
 
 <form action="/hosting/search">
 <input size="40" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 

<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 




<script type="text/javascript">
 function starClick(el, project_name) {
 _CS_toggleStar(
 el,
 {'scope': 'projects',
 'user': '_CURRENT_USER',
 'item': project_name
 },
 true,
 '',
 '',
 '',
 'star_count-' + project_name,
 'plural-' + project_name);
 }
</script>
<table class="mainhdr" width="100%" border="0" cellspacing="0" cellpadding="0">
<tr><td width="100%"></td>

 <td style="padding:3px 1em; font-weight:normal; white-space:nowrap;">
 Results 1 - 10 of 980
 </td>

</tr>
</table>
<br/>
<div id="serp">

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/th9x/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/th9x/logo?cct=1396292400.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 1)"
 href="/p/th9x/" style="font-size:medium">th9x &nbsp;- &nbsp;Alternative Firmware for 9-ch FlySky RC-Control</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Dec 14, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-th9x">67</span>
 </span><br/>
 
 
 This project is an alternative firmware for the 9 channel rc-control FS-TH9X manufactured by FlySky (other brand names: Turnigy 9X / Eurgle 9X / iMax 9X)


    Documentation (engl)
    Dokumentati...<br/>
 
 <span class="labels">
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/xe-core/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/xe-core/logo?cct=1396292400.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 2)"
 href="/p/xe-core/" style="font-size:medium">xe-core &nbsp;- &nbsp;MOVE TO GITHUB!</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Last 30 days
 &nbsp;-&nbsp;
 Stars: <span id="star_count-xe-core">85</span>
 </span><br/>
 
 
 
이 SVN 저장소 및 이슈트래커는 더 이상 사용되지 않습니다.

모든 프로젝트를 Github로 이동했습니다.

This repository and issue tracker is no longer available.
...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:XpressEngine">XpressEngine</a>, 
 
 <a href="/hosting/search?q=label:XE">XE</a>, 
 
 <a href="/hosting/search?q=label:Core">Core</a>, 
 
 <a href="/hosting/search?q=label:PHP">PHP</a>, 
 
 <a href="/hosting/search?q=label:CMS">CMS</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/additional-pipes/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 3)"
 href="/p/additional-pipes/" style="font-size:medium">additional-pipes &nbsp;- &nbsp;Additional pipes for BuildCraft</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: May 13, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-additional-pipes">12</span>
 </span><br/>
 
 
 
   Item Teleport Pipe
   Liquid Teleport Pipe
   Energy Teleport Pipe
   Distribution Pipe
   Advanced Wooden Pipe (Includes a inclusion/exclusion list)


   Add Portal type effects
   ?Reds...<br/>
 
 <span class="labels">
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/tubepress/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 4)"
 href="/p/tubepress/" style="font-size:medium">tubepress &nbsp;- &nbsp;Modern video galleries for the web</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-tubepress">44</span>
 </span><br/>
 
 
 
<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:youtube">youtube</a>, 
 
 <a href="/hosting/search?q=label:wordpress">wordpress</a>, 
 
 <a href="/hosting/search?q=label:xml">xml</a>, 
 
 <a href="/hosting/search?q=label:php">php</a>, 
 
 <a href="/hosting/search?q=label:gdata">gdata</a>, 
 
 <a href="/hosting/search?q=label:media">media</a>, 
 
 <a href="/hosting/search?q=label:video">video</a>, 
 
 <a href="/hosting/search?q=label:blogging">blogging</a>, 
 
 <a href="/hosting/search?q=label:plugin">plugin</a>, 
 
 <a href="/hosting/search?q=label:vimeo">vimeo</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/capstone-1/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 5)"
 href="/p/capstone-1/" style="font-size:medium">capstone-1 &nbsp;- &nbsp;Capstone1</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Apr 23, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-capstone-1">4</span>
 </span><br/>
 
 
 This is a website allow students evaluate their teachers and teachers can see this feedback and improve themselves.<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Academic">Academic</a>, 
 
 <a href="/hosting/search?q=label:English">English</a>, 
 
 <a href="/hosting/search?q=label:ASP.net">ASP.net</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/bm2/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 6)"
 href="/p/bm2/" style="font-size:medium">bm2 &nbsp;- &nbsp;BombusMod</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Jun 02, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-bm2">59</span>
 </span><br/>
 
 
 =Code moved to 
<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:jabber">jabber</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/sila-masjid/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 7)"
 href="/p/sila-masjid/" style="font-size:medium">sila-masjid &nbsp;- &nbsp;proyek pembuatan sistem informasi dan pelaporan keuangan masjid</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Jul 11, 2012
 &nbsp;-&nbsp;
 Stars: <span id="star_count-sila-masjid">6</span>
 </span><br/>
 
 
 Proyek ini merupakan program berbasis web dan dikembangkan dengan menggunakan bahasa pemrograman java. Hasil dari proyek ini adalah sebuah sistem berbasis web yang dapat digunakan untuk pembukuan keua...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Accounting">Accounting</a>, 
 
 <a href="/hosting/search?q=label:Java">Java</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/autoproxy/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/autoproxy/logo?cct=1396292400.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 8)"
 href="/p/autoproxy/" style="font-size:medium">autoproxy &nbsp;- &nbsp;the smart firefox proxy management add-on</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-autoproxy">268</span>
 </span><br/>
 
 
 Install: 

Subscribe: 

Develop: 

Issue: 


AutoProxy - the firefox proxy management add-on

What&#39;s New

Version 0.4 beta1 released at 12, Aug, 2010.



More details see below.


...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Firefox">Firefox</a>, 
 
 <a href="/hosting/search?q=label:add-on">add-on</a>, 
 
 <a href="/hosting/search?q=label:proxy">proxy</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/medtecs-bug-tracker/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/medtecs-bug-tracker/logo?cct=1396292400.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 9)"
 href="/p/medtecs-bug-tracker/" style="font-size:medium">medtecs-bug-tracker &nbsp;- &nbsp;MedTecs Bug Tracker</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Dec 10, 2012
 &nbsp;-&nbsp;
 Stars: <span id="star_count-medtecs-bug-tracker">1</span>
 </span><br/>
 
 
 MedTecs Bug Tracker Reporting Tool<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:erp">erp</a>, 
 
 <a href="/hosting/search?q=label:Accounting">Accounting</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/panoramagl/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/panoramagl/logo?cct=1396292400.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 10)"
 href="/p/panoramagl/" style="font-size:medium">panoramagl &nbsp;- &nbsp;Panorama viewer library for iPhone, iPad and iPod touch</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-panoramagl">91</span>
 </span><br/>
 
 
 


PanoramaGL library was the first open source library in the world to see panoramic views on the  iPhone and iPod Touch.

The supported features in version 0.1 are:

- Run on iPhone, iPod Tou...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Panorama">Panorama</a>, 
 
 <a href="/hosting/search?q=label:Panoramic">Panoramic</a>, 
 
 <a href="/hosting/search?q=label:360">360</a>, 
 
 <a href="/hosting/search?q=label:iPodTouch">iPodTouch</a>, 
 
 <a href="/hosting/search?q=label:iPhone">iPhone</a>, 
 
 <a href="/hosting/search?q=label:View">View</a>, 
 
 <a href="/hosting/search?q=label:Source">Source</a>, 
 
 <a href="/hosting/search?q=label:Code">Code</a>, 
 
 <a href="/hosting/search?q=label:Apache">Apache</a>, 
 
 <a href="/hosting/search?q=label:License">License</a>, 
 
 <a href="/hosting/search?q=label:Open">Open</a>, 
 
 <a href="/hosting/search?q=label:iPad">iPad</a>, 
 
 <a href="/hosting/search?q=label:ObjectivE-C">ObjectivE-C</a>, 
 
 <a href="/hosting/search?q=label:iOS">iOS</a>, 
 
 <a href="/hosting/search?q=label:OpenGL">OpenGL</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 </div>
 <div style="clear:both">&nbsp;</div>


<div style="text-align:center; font-size:medium; white-space:nowrap; margin-top:1em;">
 
 
 
	1
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=10">2</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=20">3</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=30">4</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=40">5</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=50">6</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=60">7</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=70">8</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=80">9</a>
 
 
 
 <a href="search?filter=0&amp;mode=&amp;start=90">10</a>
 
 
 <a href="search?filter=0&amp;mode=&amp;start=10">Next <b>&rsaquo;</b></a>
</div>
<br/>
<br/>
<div style="background:#F0F7F9; border-top:1px solid #6B90DA; border-bottom:1px solid #6B90DA; text-align:center;">
 <br/>
 <form action="/hosting/search">
 <input type="text" size="30" name="q" value="">
 <input type="hidden" name="mode" value="">
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 <br/>
</div>
<script type="text/javascript">
function clk(el, cd) {
 if (document.images) {
 (new Image()).src = "/hosting/search?sa=T&ct=res&cd=" + cd + "&url="
 + escape(el.href) + "&q=";
 }
 return true;
}
</script>
 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

