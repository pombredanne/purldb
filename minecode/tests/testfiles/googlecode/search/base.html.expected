[
  [
    "http://code.google.com/hosting/search?q=label:360", 
    "http://code.google.com/hosting/search?q=label:academic", 
    "http://code.google.com/hosting/search?q=label:accounting", 
    "http://code.google.com/hosting/search?q=label:add-on", 
    "http://code.google.com/hosting/search?q=label:apache", 
    "http://code.google.com/hosting/search?q=label:asp.net", 
    "http://code.google.com/hosting/search?q=label:blogging", 
    "http://code.google.com/hosting/search?q=label:cms", 
    "http://code.google.com/hosting/search?q=label:code", 
    "http://code.google.com/hosting/search?q=label:core", 
    "http://code.google.com/hosting/search?q=label:english", 
    "http://code.google.com/hosting/search?q=label:erp", 
    "http://code.google.com/hosting/search?q=label:firefox", 
    "http://code.google.com/hosting/search?q=label:gdata", 
    "http://code.google.com/hosting/search?q=label:ios", 
    "http://code.google.com/hosting/search?q=label:ipad", 
    "http://code.google.com/hosting/search?q=label:iphone", 
    "http://code.google.com/hosting/search?q=label:ipodtouch", 
    "http://code.google.com/hosting/search?q=label:jabber", 
    "http://code.google.com/hosting/search?q=label:java", 
    "http://code.google.com/hosting/search?q=label:license", 
    "http://code.google.com/hosting/search?q=label:media", 
    "http://code.google.com/hosting/search?q=label:objective-c", 
    "http://code.google.com/hosting/search?q=label:open", 
    "http://code.google.com/hosting/search?q=label:opengl", 
    "http://code.google.com/hosting/search?q=label:panorama", 
    "http://code.google.com/hosting/search?q=label:panoramic", 
    "http://code.google.com/hosting/search?q=label:php", 
    "http://code.google.com/hosting/search?q=label:plugin", 
    "http://code.google.com/hosting/search?q=label:proxy", 
    "http://code.google.com/hosting/search?q=label:source", 
    "http://code.google.com/hosting/search?q=label:video", 
    "http://code.google.com/hosting/search?q=label:view", 
    "http://code.google.com/hosting/search?q=label:vimeo", 
    "http://code.google.com/hosting/search?q=label:wordpress", 
    "http://code.google.com/hosting/search?q=label:xe", 
    "http://code.google.com/hosting/search?q=label:xml", 
    "http://code.google.com/hosting/search?q=label:xpressengine", 
    "http://code.google.com/hosting/search?q=label:youtube"
  ], 
  [
    {
      "url": "http://code.google.com/p/th9x/", 
      "last_updated": "2013-12-14T00:00:00+00:00", 
      "stars": 67
    }, 
    {
      "url": "http://code.google.com/p/xe-core/", 
      "last_updated": "2014-01-03T00:00:00+00:00", 
      "stars": 85
    }, 
    {
      "url": "http://code.google.com/p/additional-pipes/", 
      "last_updated": "2013-05-13T00:00:00+00:00", 
      "stars": 12
    }, 
    {
      "url": "http://code.google.com/p/tubepress/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 44
    }, 
    {
      "url": "http://code.google.com/p/capstone-1/", 
      "last_updated": "2013-04-23T00:00:00+00:00", 
      "stars": 4
    }, 
    {
      "url": "http://code.google.com/p/bm2/", 
      "last_updated": "2013-06-02T00:00:00+00:00", 
      "stars": 59
    }, 
    {
      "url": "http://code.google.com/p/sila-masjid/", 
      "last_updated": "2012-07-11T00:00:00+00:00", 
      "stars": 6
    }, 
    {
      "url": "http://code.google.com/p/autoproxy/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 268
    }, 
    {
      "url": "http://code.google.com/p/medtecs-bug-tracker/", 
      "last_updated": "2012-12-10T00:00:00+00:00", 
      "stars": 1
    }, 
    {
      "url": "http://code.google.com/p/panoramagl/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 91
    }
  ]
]